### PR TITLE
New package: GNU.Icecat version 91.4.1

### PR DESCRIPTION
**Pull request by Dfmaaa**
GNU Icecat for Windows

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40713)

### CODE REVIEW

File: manifests/g/GNU/Icecat/91.4.1/GNU.Icecat.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+PackageIdentifier: GNU.Icecat
```
Comment (Trenly) [inline manifests/g/GNU/Icecat/91.4.1/GNU.Icecat.yaml:1]:
```suggestion
PackageIdentifier: muslayev.Icecat
```

File: manifests/g/GNU/Icecat/91.4.1/GNU.Icecat.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageVersion: 91.4.1
+PackageName: icecat
+Publisher: GNU.org
```
Comment (Trenly) [inline manifests/g/GNU/Icecat/91.4.1/GNU.Icecat.yaml:4]: This must match what is shown in control panel
```suggestion
Publisher: Mozilla
```

File: manifests/g/GNU/Icecat/91.4.1/GNU.Icecat.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+PackageIdentifier: GNU.Icecat
+PackageVersion: 91.4.1
+PackageName: icecat
```
Comment (Trenly) [inline manifests/g/GNU/Icecat/91.4.1/GNU.Icecat.yaml:3]:
```suggestion
PackageName: IceCat
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\g\GNU\Icecat\91.4.1', 
expected 'manifests\m\muslayev\Icecat\91.4.1'
```

**Pull request closed without merging**